lib/request.js

```diff
@@ -54,6 +105,21 @@ Object.defineProperties(Request.prototype, {
     get () {
       return this.raw.connection
     }
+  },
+  ip: {
+    get () {
+      return this.connection.remoteAddress
+    }
+  },
+  ips: {
+    get () {
+      return undefined
+    }
+  },
```

Comment:
Since this is always returning undefined... should I even bother adding the getter?

Reply:
no

Reply:
I believe this is only set when an `x-forwarded-for` header is present.

Reply:
> I believe this is only set when an x-forwarded-for header is present.

and `trustProxy` is truthy.

Reply:
So that codepath should still be able to set this property.